docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,378 @@
+    machineconfiguration.openshift.io/role: worker
+spec:
+  osImageURL: ''
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+        - name: create-mountpoint-var-config.service
+          enabled: true
+          contents: |
+            [Unit]
+            Description=Create mountpoint /var/config
+            Before=kubelet.service
+
+            [Service]
+            ExecStart=/bin/mkdir -p /var/config
+
+            [Install]
+            WantedBy=var-config.mount
+
+        - name: var-config.mount
+          enabled: true
+          contents: |
+            [Unit]
+            Before=local-fs.target
+            [Mount]
+            Where=/var/config
+            What=/dev/disk/by-label/config-2
+            [Install]
+            WantedBy=local-fs.target
+```
+
+### Enabling the No-IOMMU feature for the RHOSP VFIO driver
+
+You can apply a machine config to your machine pool that enables the No-IOMMU feature for the Red Hat OpenStack Platform (RHOSP) virtual function I/O (VFIO) driver. The RHOSP vfio-pci driver requires this feature.
+
+```yaml
+kind: MachineConfig
+apiVersion: machineconfiguration.openshift.io/v1
+metadata:
+  name: 99-vfio-noiommu
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  osImageURL: ''
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - filesystem: root
+        path: "/etc/modprobe.d/vfio-noiommu.conf"
+        contents:
+          source: data:text/plain;charset=utf-8;base64,b3B0aW9ucyB2ZmlvIGVuYWJsZV91bnNhZmVfbm9pb21tdV9tb2RlPTEK
+          verification: {}
+        mode: 0644
+```
+
+If you need to configure your deployment for real-time running or low latency, see the [performance addon operator documentation](performance-addon-operator), or you plan to use a technology such as DPDK that requires the use of [hugepages](huge-pages-perf-addon).
+
+Once your OpenShift control plane is running, you will need to install the SR-IOV Network Operator. To do so, you will need access to an account on your OpenShift cluster with `cluster-admin` privileges. Once you are logged into that account, follow [these steps](sriov-operator) to install the Operator. Then, [configure your SR-IOV network device](configure-sriov-network-device).
+
+## Creating SR-IOV Networks for Worker Nodes
+
+The SR-IOV networks that are attached to worker nodes need to be created before creating the nodes. Please reference the example of how to create radio and uplink provider networks in OpenStack.
+
+```sh
+# Create Networks
+openstack network create radio --provider-physical-network radio --provider-network-type vlan --provider-segment 120
+openstack network create uplink --provider-physical-network uplink --provider-network-type vlan --external
+
+# Create Subnets
+openstack subnet create --network radio --subnet-range <radio_network_subnet_range> radio
+openstack subnet create --network uplink --subnet-range <uplink_network_subnet_range> uplink
+```
```

Comment:
It would be nice if we could create this thorugh the openshift cluster (through yaml file or something) I guess for future enhancent

Reply:
+1 lets wait for the upstream CAPO merge to consider that